gdk-pixbuf/Cargo.toml

```diff
@@ -13,6 +13,7 @@ build = "build.rs"
 exclude = [
     "gir-files/*",
 ]
+edition = "2018"
```

Comment:
Randomly commenting here but you have to clean up all the non-sys Cargo.toml :)

```diff
-gdk-pixbuf-sys = { path = "sys" }
+ffi = { package = "gdk-pixbuf-sys", path = "sys" }
 glib = { path = "../glib" }
-glib-sys = { path = "../glib/sys" }
-gobject-sys = { path = "../glib/gobject-sys" }
-gio-sys = { path = "../gio/sys" }
```

This makes the `extern crate gdk_pixbuf_sys as ffi;` in lib.rs unnecessary, and you reference all the other sys crates via their non-sys equivalent anyway... `gio::ffi` instead of `gio_sys`.

Reply:
We already talked about that. Considered the vast amount of changes, I don't intend to do it in the same PR. :)

Reply:
It's a quite easy change that can be automated with sed. @bilelmoussaoui and I did that for our crates already when updating things

Reply:
https://github.com/gtk-rs/gtk-rs/issues/164